### DISCUSSION TITLE
Actual license?

### DISCUSSION THREAD

**Discussion by eg3r** [BepInEx/BepInEx#649] [General]
Can you guys verify what the actual license is? Here on git the LGPL-2.1 license is used, while the official docs say:

"BepInEx is fully open source under the MIT license." ( https://docs.bepinex.dev/index.html ) 

Someone forgot to change it somehwere? Did I miss a discussion somewhere? If so please show me the way. Thanks.

**Comment by js6pak** [comment]
v5 is MIT, v6/master is LGPL. You have linked to v5 docs, v6 docs are under http://docs.bepinex.dev/master/

**Comment by eg3r** [comment]
Thank you for the information.
